core/src/commonMain/kotlin/com/ustadmobile/core/contentjob/ContentPluginManager.kt

```diff
@@ -0,0 +1 @@
+
```

Comment:
Not sure why the diff does not show here... anyhow, let's add a  brief comment for the class

/* 
 * ContentPluginManager is a simple Singleton that maintains a list of all available ContentPlugins. It has convenience functions that iterate over all available plugins.
 */

Reply:
ContentPluginManager should throw an exception when a duplicate id is used. This could be done using an init blcok e.g.
```
val pluginDuplicates = pluginList.filter { plugin -> 
    pluginList.count { it.pluginId == plugin.pluginId } > 1 
}
if(pluginDuplicates.isNotEmpty()) {
    throw IllegalArgumentException("Duplicate pluginIds in: ${pluginDuplicates.joinToString()}")
}
```